src/rolls.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+function Rolls() {
+  this.currentGame = [];
+  this.score = 0;
+  this.frameNum = 1;
+  this.rollOneTaken = false;
+  this.rollOneScore = 0;
+  this.rollTwoScore = 0;
+  this.frameScore = 0;
+}
+
+var strike = 10
```

Comment:
these should not be defined at the top level; they are essentially now global variables.